cmd/template/cluster/flag.go

```diff
@@ -24,6 +24,21 @@ const (
 	flagAWSEKS                = "aws-eks"
 	flagAWSControlPlaneSubnet = "control-plane-subnet"
 
+	flagAWSRegion                      = "region"
+	flagAWSRole                        = "role"
+	flagNetworkAZUsageLimit            = "az-usage-limit"
+	flagNetworkVPCCidr                 = "vpc-cidr"
+	flagBastionInstanceType            = "bastion-instance-type"
+	flagBastionReplicas                = "bastion-replicas"
+	flagControlPlaneInstanceType       = "control-plane-instance-type"
+	flagAWSMachinePoolMinSize          = "machine-pool-min-size"
+	flagAWSMachinePoolMaxSize          = "machine-pool-max-size"
+	flagAWSMachinePoolName             = "machine-pool-name"
+	flagAWSMachinePoolAZs              = "machine-pool-azs"
+	flagAWSMachinePoolInstanceType     = "machine-pool-instance-type"
+	flagAWSMachinePoolRootVolumeSizeGB = "machine-pool-root-volume-size-gb"
+	flagAWSMachinePoolCustomNodeLabels = "machine-pool-custom-node-labels"
```

Comment:
It sounds like some of these can be shared across providers, like `bastion-instance-type` or `bastion-replicas` or `control-plane-instance-type` or even `region` (if I agree on using that term for all providers). Any reason not to share them?

Reply:
the naming terminology is not consistent across providers so I am not sure how good UX is to mix them, on was we have instance type, on OpenStack there is flavor, Not sure how would someone who never worked with  Openstack created if you need to select bastion flavor for AWS

Reply:
the whole flag is a mess in my opinion and I don't know a good  approach how to get it straight and have a good UX for it

Reply:
I think sorting all flags across all provider should not be scope fo this PR, but rather some big story for KaaS sync